node/src/actors/chain_manager/mod.rs

```diff
@@ -170,7 +170,7 @@ pub struct ChainManager {
     /// Blockchain state data structure
     chain_state: ChainState,
     /// Backup for ChainState
```

Comment:
Maybe we could add why we need the backup (for resetting the state after a reorganization)